api/src/migrations/remove-user-language.js

```diff
@@ -0,0 +1,37 @@
+const db = require('../db');
+const BATCH_SIZE = 100;
+
+const removeLanguage = (docs) => {
+  docs.forEach(doc => delete doc.language);
```

Comment:
I just realized we have another regression because of this, in webapp: 

![image](https://user-images.githubusercontent.com/35681649/121675579-3f530580-cabc-11eb-9ed2-ebbcc9933daf.png)

The language dropdown used to get the value from the user settings document: https://github.com/medic/cht-core/blob/master/webapp/src/ts/modals/edit-user/edit-user.component.ts#L34 . Now it's always blank (duh). 
We should change this to get the language from the cookie.

Reply:
Okay, I am pretty sure I have got this fixed correctly (full disclosure: I never touched Angular before last week...).  I also just completely removed the call to the user settings from the LanguageService since it seemed pointless now.